cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -138,7 +108,7 @@ def test_menuwidget_crud(appliance):
             "Services / Catalogs": fauxfactory.gen_alphanumeric(),
             "Cloud Intel / Dashboard": fauxfactory.gen_alphanumeric(),
         },
-        visibility="<To All Users>"
+        visibility="<To All Users>",
     )
```

Comment:
Do you need this change? I mean this is completely different test case.

Reply:
I was blackifying the file, the changes were not that huge, so I let them be. Those are blackified changes.